getpaid/models.py

```diff
@@ -1,5 +1,7 @@
 import sys
 from datetime import datetime
+
+from django.apps import apps
```

Comment:
This is detail but `apps` is also a module in current dir, would it make sense to just import what we need instead of importing module?